dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/FATSuite.java

```diff
@@ -19,13 +19,12 @@
 import com.ibm.ws.jca.fat.app.DependantApplicationTest;
 import com.ibm.ws.jca.fat.app.JCATest;
 
-import componenttest.custom.junit.runner.AlwaysPassesTest;
 import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.JakartaEE9Action;
```

Comment:
were you intending to apply the JakartaEE9 action with this bucket?

Reply:
Yes, but likely in a future PR.  I was testing it out, but will likely go through these one at a time and resolve issues along the way. :)